llvmlite/ir/values.py

```diff
@@ -52,6 +52,24 @@ def bitcast(self, typ):
                                                typ)
         return FormattedConstant(typ, op)
 
+    def ptrtoint(self, typ):
+        """
+        Cast this pointer constant to the given integer type.
+        """
+        if not isinstance(typ, types.IntType):
+            raise TypeError(
+                "can only ptrtoint() to integer type, not '%s'"
+                % (typ,))
+        if not isinstance(self.type, types.PointerType):
+            raise TypeError(
+                "can only call inttoptr() on pointer type, not '%s'"
+                % (typ,))
```

Comment:
```suggestion
                % (self.type,))
```

Reply:
Thanks for catching this. Fixed in updated patch.

Reply:
No problem, thanks for the fix.